### PR TITLE
Add TSDoc for all the classes and methods in Core

### PR DESCRIPTION
**Pull request by gkhunger-sf**

### CODE REVIEW

File: packages/core/src/base/staticClient.ts [hunk 1]
```diff
@@ -47,11 +62,12 @@ export async function getObjectFromResponse(
...
- * @param header - target header
- * @param headers - list to search from
- *
- * @returns header from the list if there is a match, the header passed otherwise
+ * 
+ * @param header - Target header
+ * @param headers - List to search from
+ * @returns Header from the list if there is a match, the passed header otherwise
+ * 
+ * @export
```
Comment (unandyala) [inline packages/core/src/base/staticClient.ts]: What is @export for? If we are not not using, I suggest to remove it

Comment (gkhunger-sf) [reply packages/core/src/base/staticClient.ts]: It is just to state that it is an exported method. It's just extra information. I saw that we were using it in some of our other doc strings and decided to use it. I don't have any strong feelings about using it or not using it. If you feel it is not very useful I can remove it.

Comment (unandyala) [reply packages/core/src/base/staticClient.ts]: does not sound very useful

Comment (gkhunger-sf) [reply packages/core/src/base/staticClient.ts]: Removed

**Pull request merged**